core/src/main/java/org/jboss/modcluster/ModClusterService.java

```diff
@@ -21,24 +21,6 @@
  */
 package org.jboss.modcluster;
 
-import java.io.IOException;
```

Comment:
Please do not change the order of the imports in an existing source files (same would apply for formatting changes) in a feature commit.

Reply:
reverted now. It was done automatically by the codestyle profile in our IDE (default Java).